package.json

```diff
@@ -32,12 +32,14 @@
     "@material-ui/core": "4.12.3",
     "@material-ui/icons": "4.11.2",
     "@material-ui/lab": "4.0.0-alpha.57",
+    "@sentry/react": "^7.1.1",
+    "@sentry/tracing": "^7.1.1",
     "assert": "^2.0.0",
     "axios": "0.24.0",
     "bignumber.js": "9.0.1",
     "buffer": "^6.0.3",
-    "crypto-browserify": "^3.12.0",
     "crypto": "npm:crypto-browserify",
+    "crypto-browserify": "^3.12.0",
```

Comment:
do we need both?

Reply:
Yes. The `crypto` is only an alias for the later one.